code/game/objects/machinery/vending/vending_types.dm

```diff
@@ -421,6 +421,23 @@
 		if(WEST)
 			pixel_x = 21
 
+/obj/machinery/vending/nanomed/tadpolemed
+	name = "Flight surgeron medical equipment dispenser"
+	desc = "Dedicated for the surgeron with wings, this humble box contains a lot for its size."
+	products = list(
+		/obj/item/reagent_containers/hypospray/autoinjector/combat = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/bicaridine = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/kelotane = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/dylovene = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/tricordrazine = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/tramadol = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/hypervene = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/inaprovaline = 0,
+		/obj/item/stack/medical/heal_pack/gauze = 2,
+		/obj/item/stack/medical/heal_pack/ointment = 2,
+		/obj/item/healthanalyzer = 1,
+		/obj/item/stack/medical/splint = 1,
+	)
```

Comment:
are you sure you only want to authorize this? What about pill bottle? This should be able to hold as much as hold vendor imo

Reply:
The previous NanotrasenMed Plus only hold peridaxon and Russian Red.

While it's true that NanotrasenMed Plus hold med rig and by extension pill bottles, it's too far of a stretch to claim that NanotrasenMed Plus holds pill bottles like a MarineMed.